### PR TITLE
demo: suspense demo

### LINKED ISSUE #90: Example for Suspense
Provide a simple and clear example to show SWR + Suspense, with [error handling](https://reactjs.org/docs/concurrent-mode-suspense.html#handling-errors) and data updates.

### PR DESCRIPTION
**Pull request by wendzhue**
close #90

This pull request adds a Suspense example.

With `<Suspense>`:

Snipaste_2019-11-09_16-22-51

With `<ErrorBoundary>`, when the application fails to get response (e.g the user is offline):

Snipaste_2019-11-09_16-19-57

### CODE REVIEW

**Comment by wendzhue** [comment]
Still, I cannot get the demo running if swr is resolved by `link:../..`. @quietshu

**Comment by quietshu** [comment]
Thank you @wendzhue! 

We're currently using the package for examples (not `link:../..` anymore), so they can be standalone projects. And also `yarn.lock` can be removed as well (it'll be cleaner) : )

File: examples/suspense/package.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "name": "suspense",
+    "version": "1.0.0",
+    "main": "index.js",
+    "license": "MIT",
+    "dependencies": {
+      "isomorphic-unfetch": "3.0.0",
+      "next": "9.1.1",
+      "swr": "^0.1.9"
```
Comment (sergiodxa) [inline examples/suspense/package.json]: You could use latest, without the lock file it will really point to the latest version of SWR

**Comment by quietshu** [comment]
Can you add `react` and `react-dom` to dependencies as well?

**Comment by wendzhue** [comment]
> Can you add `react` and `react-dom` to dependencies as well?

Sure.

**Pull request merged**